src/capi_frontend/server_settings.hpp

```diff
@@ -125,6 +131,14 @@ struct EmbeddingsGraphSettingsImpl {
     std::string pooling = "CLS";
 };
 
+struct TextToSpeechGraphSettingsImpl {
+    uint32_t numStreams = 1;
```

Comment:
We have num streams in exportSettings.pluginConfig:
https://github.com/openvinotoolkit/model_server/pull/3765/files#diff-a434039244a1d827b2658d34fa985eaa85ea35fa9f6a6c0a94745e4732c31165R101